portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,190 @@
+@import url('https://fonts.googleapis.com/css?family=Major Mono Display');
+@import url('https://fonts.googleapis.com/css?family=Codystar');
+@import url('https://fonts.googleapis.com/css?family=Nothing You Could Do');
+
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+  margin-left: 15%;
+  margin-right: 15%;
 }
 
 #greeting-container {
   margin-top: 20px;
+  
+}
+
+#TechCorner {
+    background-color: #F0EDE5;
+    text-align: right;
+}
+
+#Contact {
+    background-color: #F0EDE5;
+    text-align: right;
+}
+
+
+img.image {
+    float: right;
```

Comment:
Flex is kind of like the modern version of float https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Reply:
I decided not to do flex because on smaller websites, I want the text to still wrap around the image, instead of being strictly half and half:)